scripts/release-notes.py

```diff
@@ -88,17 +108,70 @@ def main(token, repo, num_commits, exclude_dependabot):
         if not pulls:
             short_sha = sha[:7]
             commit_url = commit['html_url']
-            print(f'* {msg} ([@{author}]({author_url}) in [{short_sha}]({commit_url}))')
+            # Check if the commit has changelog label
+            commit_labels = get_pull_request_labels(token, args.repo, pull_id)
```

Comment:
if there are `not pulls` where are you getting `pull_id`?

Reply:
Correct actually. Now, I just threw all commits without pull_ids to the `UNCATEGORISED` group.